products/sle12/profiles/anssi_bp28_minmal.profile

```diff
@@ -0,0 +1,26 @@
+documentation_complete: true
```

Comment:
@teacup-on-rockingchair Is the intention to name the profile 'minmal`?
I see that `sle15` also got a profile named like that: `anssi_bp28_minmal`

Reply:
This intention was to follow the scheme for rhel8, and once we have all the rules covered will add also high, enhanced and intermediate.

Reply:
Anyway, I can merge this as is, and the typo can be corrected later.